Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3
```

Comment:
Can we use Alpine here? `python:3-alpine`
That would allow us to create the smallest image possible. Otherwise, it is based on Ubuntu and will already come with a few hundred megabytes of stuff included.